src/components/Home.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { loading } from '../features/country/countrySlice';
+import IconArrowRightCircle from './IconArrowRightCircle';
+
+const Home = () => {
```

Comment:
- [ ] On the homepage, kindly add the hero section as in the design from Behance, with information describing what the app is intended for. This helps visitors get a better understanding of what information the app offers.

  ![image](https://github.com/kerlos-bahaa/React-capstone/assets/81468675/10e5e6d6-7bf6-420d-960b-e2fc1b8bd266)